src/pages/FilesPage/FilesTabs/DatasetsPane/DatasetsPane.tsx

```diff
@@ -29,18 +30,27 @@ export const DatasetsPane: FC = function DatasetsPane() {
         setDatasets(datasets);
         setIsLoading(false);
       })
-      .catch(() => {
+      .catch(res => {
         setIsError(true);
         setIsLoading(false);
+        setErrorMessage(res);
       });
   }, []);
 
   if (isError) {
     return (
       <Result
         status="error"
-        subTitle="Please, try again"
-        title="Failed to load datasets"
+        subTitle={
+          errorMessage.includes('NoneType')
```

Comment:
Its a kind of magic. don't do so.
create `const is?????Error = errorMessage.includes('NoneType')`